spark/v2.4/spark/src/test/java/org/apache/iceberg/spark/source/TestIcebergSourceTablesBase.java

```diff
@@ -783,7 +783,7 @@ public void testSnapshotsTable() {
             .set("summary", ImmutableMap.of(
                 "added-records", "1",
                 "added-data-files", "1",
-                "changed-partition-count", "1",
+                "changed-partition-count", "0",
```

Comment:
This depends on whether we consider the root to be a partition.

I think we should leave this as-is because there isn't a compelling reason to set this to 0 partitions, and that is a behavior change. We should suppress the root partition summary since it is identical to the table summary, but we shouldn't change this.

Reply:
I agree that the root should be considered a partition of its own. Suppressing the root partition summary is sufficient.

Will update 👍